gradle.properties

```diff
@@ -12,9 +12,9 @@
 #
 #
 
-group=com.huawei.cloud
-version=0.11.0-SNAPSHOT
+group=org.eclipse.edc.huawei
+version=0.12.0-SNAPSHOT
```

Comment:
this version should not change otherwise the release process for 0.11.0 will fail, on main it could be changed (it should be already point to 0.12.0-SNAPSHOT)